pyomo/common/backports.py

```diff
@@ -1,22 +0,0 @@
-#  ___________________________________________________________________________
-#
-#  Pyomo: Python Optimization Modeling Objects
-#  Copyright (c) 2008-2022
-#  National Technology and Engineering Solutions of Sandia, LLC
-#  Under the terms of Contract DE-NA0003525 with National Technology and
-#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
-#  rights in this software.
-#  This software is distributed under the 3-clause BSD License.
-#  ___________________________________________________________________________
-
-
-# backport of contextlib.nullcontext for supporting Python < 3.7
-class nullcontext(object):
```

Comment:
I would argue *against* deleting this file, as that breaks any/all potential dependencies.  Instead, this file should just
```python
from contextlib import nullcontext
```